src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+@import url(./stars.scss);
+$color-blue: #00acdc;
+$color-gray: #616070;
+$color-black: #060b35;
+  
+.card {
+  font-family: "Roboto", sans-serif;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+
+  &__wrapper {
+    display: flex;
+    flex-direction: column;
+    padding: 32px 16px 16px;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__title-wrapper {
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-gray;
+    margin-bottom: 16px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-black;
+    margin-bottom: 4px;
+
+    &-label {
+      font-weight: 400;
+    }
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 10px;
+    color: $color-black;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    font-weight: 400;
+    font-size: 12px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-label {
+      line-height: 18px;
+      color: $color-gray;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    min-width: 166px;
+    min-height: 40px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    background: #00acdc;
```

Comment:
accent color from variables

Reply:
Yep, this color is used in many places. Move it to variables.

Reply:
it is

Reply:
done